### PR TITLE
Updated GraphQL-RelaySpecification.md to be more readable in "Getting started" part

### PR DESCRIPTION
**Pull request by liesislukas**
made it more clear to read

### CODE REVIEW

File: docs/GraphQL-RelaySpecification.md [hunk 1]
```diff
@@ -9,10 +9,13 @@ next: graphql-object-identification
 
 # Getting Started
 
-The three core assumptions that Relay makes on a GraphQL server are that
-it provides a mechanism for refetching an object, a description of how
-to page through connections, and a structure around mutations to make
-them predictable. This example demonstrates all three of these assumptions.
+The three core assumptions that Relay makes on a GraphQL server are that it provides:
```
Comment (wincent) [inline docs/GraphQL-RelaySpecification.md]: Can we make this "makes _about_ a GraphQL server" instead?

**Comment by wincent** [comment]
Thanks for this. Would you mind updating the title to something more descriptive than "Update GraphQL-RelaySpecification.md"?

**Comment by liesislukas** [comment]
looks good?

File: docs/GraphQL-RelaySpecification.md [hunk 2]
```diff
@@ -9,10 +9,13 @@ next: graphql-object-identification
...
+
+1. Mechanism for refetching an object
+2. Description of how to page through connections
```
Comment (josephsavona) [inline docs/GraphQL-RelaySpecification.md]: nit: "A mechanism..." + ending period (lines 14-15).

**Comment by josephsavona** [comment]
@liesislukas looks good, see our feedback.

**Comment by steveluscher** [comment]
Ping @liesislukas; we'd love to get this in. Do you have time to update the PR with @josephsavona's comments?

**Comment by wincent** [comment]
@liesislukas, I've added another commit on top of this one here to address the issues raised, so I am going to close out this pull request.

Thanks for your contribution!

**Pull request closed without merging**